sdk/notificationhubs/notification-hubs/samples-dev/createInstallation.fcmLegacy.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how the createOrUpdateInstallation() method can be used to create or overwrite an
+ * installation in place.
+ *
+ * See https://docs.microsoft.com/azure/notification-hubs/notification-hubs-push-notification-registration-management
+ * to learn about installations.
+ *
+ *
+ * @summary Demonstrates how to create or overwrite an installation using Azure Notification Hubs
+ * @azsdk-weight 100
+ */
+
+import { createClientContext } from "@azure/notification-hubs/client";
+import { createFcmLegacyInstallation } from "@azure/notification-hubs/models/installation";
+import { createOrUpdateInstallation } from "@azure/notification-hubs/client/createOrUpdateInstallation";
+import { v4 as uuid } from "uuid";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// Define connection string and hub name
+const connectionString = process.env.NOTIFICATIONHUBS_CONNECTION_STRING || "<connection string>";
+const hubName = process.env.NOTIFICATION_HUB_NAME || "<hub name>";
+
+// Define message constants
+const DUMMY_REGISTRATION = "bk3RNwTe3H0:CI2k_HHwgIpoDKCIZvvDMExUdFQ3P1";
```

Comment:
I wonder why this is not an env var instead?

Reply:
If the customer wants to run something just to see if it can communicate with the back end, then a hard coded dummy token works fine.  This will read from the environment if it's not there.